stable/atlantis/templates/statefulset.yaml

```diff
@@ -72,6 +72,9 @@ spec:
               exec:
                 command: ["/bin/sh", "-c", "cp /etc/secret-gitconfig/gitconfig /home/atlantis/.gitconfig && chown atlantis /home/atlantis/.gitconfig"]
           {{- end}}
+          {{- if .Values.command }}
+          command: {{ .Values.command }}
```

Comment:
`command` takes an array so this should be
```
command:
{{- range .Values.command }}
- {{ . }}
{{- end }}
```

Reply:
@lkysow thank you. done all suggestions.